run_tests.py

```diff
@@ -0,0 +1,58 @@
+# This script is meant to carry out an automated test on all examples
+# of Nabla and log the console output of each to a text file which is
+# created and placed in the same directory as the executable of each example
+
+
+import glob, os, subprocess, time
+
+
+# walk into the examples_tests directory
+os.chdir("examples_tests")
+
+# list containing indices of each example to test
+# (I'm supposed to use list comprehension here but for some weird reason 'glob' refuse to work when I use it :\)
+indices = [1,2,3]
+
+# format each element of 'indices' to 2 digit
+def format_list():
+	for i in range(len(indices)):
+		indices[i] = str(indices[i])
+		indices[i] = '0' + indices[i]
+		if len(indices[i]) > 2:
+			indices[i] = indices[i].replace(indices[i][0], '', 1)
+
+
+def run_tests():
+	for i in range(len(indices)):
+		for fn in glob.glob(indices[i] + ".*"):
+		    print("\n", fn)
+		    os.chdir(fn)
+		    os.chdir("bin")
+
+		for fn in glob.glob("*.exe"):

```

Comment:
We don't need every exe in `bin/` we just need the release version and it has the format of `targetname.exe` and NOT `targetname_d.exe` or `targetname_rwdi.exe`
and targetname is `tolower(examplefoldername without the number and the dot at the beginning)` please verify @AnastaZIuk 

Also consider as example failure and log appropriate message when this exe is not found as if the example failed

Reply:
configuration*, my phone autocorrection is ugly